examples/quickstart-python/quickstart.py

```diff
@@ -17,117 +17,120 @@
 
 if code.status_code == 204:
     print("Password is set!")
 else:
-    print("Unnable to set Password")
+    print("Unable to set Password")
 
 # Now we can start the client application
 print("You could now start the Hono Command Line Client in another terminal to consume messages from devices:")
 print()
-cmd = f'java -jar hono-cli-2.*-exec.jar app --sandbox --amqp consume --tenant={tenantId}'
+cmd = f"java -jar hono-cli-2.*-exec.jar app --sandbox --ca-file {CA_FILE} consume --tenant={tenant_id}"
 print(cmd)
 print()
 
 
 # input("Press Enter to continue...")
 
-class AmqpHandler(MessagingHandler):
-    """
-    Handler for "northbound side" where Messages are received
-    via AMQP.
-    """
-    def __init__(self, server, address):
-        super(AmqpHandler, self).__init__()
-        self.server = server
-        self.address = address
-
-    def on_start(self, event):
-        # the password used below is the Sandbox server's password for the default consumer
-        # as documented on the Hono website
-        conn = event.container.connect(self.server, user="consumer@HONO", password="verysecret") # nosec hardcoded_password_funcarg
-        event.container.create_receiver(conn, self.address)
-
-    def on_connection_error(self, event):
-        print("Connection Error")
-
-    def on_link_error(self, event):
-        print("Link Error")
-
-    def on_message(self, event):
-        print("Got a message:")
-        print(event.message.body)
-
-
-# Prepare the container
-uri = f'amqp://{amqpNetworkIp}:15672'
-address = f'telemetry/{tenantId}'
-print("Using source: " + uri)
-print("Using address: " + address)
-container = Container(AmqpHandler(uri, address))
-
-# run container in separate thread
-print("Starting (north bound) AMQP Connection...")
-thread = threading.Thread(target=lambda: container.run(), daemon=True)
-thread.start()
-
-# Give it some time to link
-time.sleep(2)
+class DownstreamApp(threading.Thread):
+    def __init__(self):
+        threading.Thread.__init__(self)
+        self.stop_event = threading.Event()
+        self._topic_name = f"hono.telemetry.{tenant_id}"
+
+    def stop(self):
+        self.stop_event.set()
+
+    def run(self):
+
+        consumer = KafkaConsumer(
+            self._topic_name,
+            bootstrap_servers=f"{HONO_SANDBOX_HOSTNAME}:9094",
+            group_id="foo",
+            max_poll_records=10,
+            ssl_cafile=CA_FILE,
+            security_protocol="SASL_SSL",
+            sasl_mechanism="SCRAM-SHA-512",
+            sasl_plain_username="hono",
+            sasl_plain_password="hono-secret")
+
+        print("created Kafka consumer ...")
+
+        print("waiting for Kafka messages")
+        msg_count = 0
+        for message in consumer:
+            print(f"received message: {message.value.decode()}")
+            msg_count += 1
+            if msg_count >= 2 or self.stop_event.is_set():
+                break
+
+        consumer.close()
+
```

Comment:
nit: Extra line break should be added here ("PEP 8: E305 expected 2 blank lines after class or function definition, found 1").